site-src/getting-started.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with Network Policy API
+
+**1. Install a Network Policy API compatible CNI**
+
+There are numerous Container Network Plugin projects that support or are actively working on
+supporting the Network Policy API. Please refer to the [implementations](implementations.md)
+doc for details on the supportability matrix.
+
+**2. Install the Network Policy API CRDs**
+
+The following commands will install the latest version of the `AdminNetworkPolicy` and 
+`BaselineAdminNetworkPolicy` objects in your K8s cluster:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/network-policy-api/master/config/crd/policy.networking.k8s.io_adminnetworkpolicies.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/network-policy-api/master/config/crd/policy.networking.k8s.io_baselineadminnetworkpolicies.yaml
+```
```

Comment:
I would also add a note or warning on: Since the master branch can have activate changes, for best results point to the released version of the API that will be stable?
example:
https://raw.githubusercontent.com/kubernetes-sigs/network-policy-api/v0.1.0/config/crd/policy.networking.k8s.io_adminnetworkpolicies.yaml
instead of master, cause that could break users if they point to master

Reply:
Even simpler this should be 

`kubectl apply -f https://github.com/kubernetes-sigs/network-policy-api/releases/download/v0.1.0/install.yaml`

Reply:
And we need to add a note to the release doc to make sure that version number is updated everytime a release is cut.

Reply:
Done